bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
https://devcenter.heroku.com/articles/buildpack-api#caching - At the end of the build, move bower_components into the $CACHE_DIR. Then before `bower install` pull the `bower_components` dir out of the cache.